src/main/java/org/tarantool/jdbc/SQLResultSet.java

```diff
@@ -318,7 +318,7 @@ public InputStream getAsciiStream(String columnLabel) throws SQLException {
     @Override
     public InputStream getUnicodeStream(int columnIndex) throws SQLException {
         String string = getString(columnIndex);
-        return string == null ? null : new ByteArrayInputStream(string.getBytes(Charset.forName("UTF-16")));
+        return string == null ? null : new ByteArrayInputStream(string.getBytes(Charset.forName("UTF-8")));
```

Comment:
Hm. There is the following [commit](d4d62e99952ddb0f43209aab6406903436cff698):

```git
commit d4d62e99952ddb0f43209aab6406903436cff698
Author: nicktorwald <nicktorwald@gmail.com>
Date:   Mon May 20 16:57:41 2019 +0700

    Change getUnicodeStream encoding from UTF-8 to UTF-16.

    The JDBC standard requires two-bytes encoding to be returned from
    getUnicodeStream.
```

Are you sure it should be reverted? If this change is intentional, then it worth to describe a reason in a commit message (I think it is not obvious) and mention this commit in your's.

Reply:
I was confused by API ResultSet.getUnicodeStream:

> Retrieves the value of the designated column in the current row of this ResultSet object as as a stream of *two-byte 3 characters*. The first byte is the high byte; the second byte is the low byte.

What does really mean two-byte 3 characters? Maybe `3` is a redundant typo like `as as a stream` just before in the sentence.  Other vendors who support Unicode stream return UTF-8.
Nevertheless, according to API, it looks like UTF-16BE. 

On the other side, PreparedStatement.setUnicodeStream says:

> The byte format of the Unicode stream must be a Java UTF-8, as defined in the Java Virtual Machine Specification.

which is a bit asymmetric to `ResultSet.getUnicodeStream`.

Reply:
Okay. I propose to revert d4d62e99952ddb0f43209aab6406903436cff698 in a separate commit and properly describe the reason of the revertion (like you done it here).